apps/wallet/src/ui/app/pages/home/transfer-coin/TransferCoinForm/StepOne.tsx

```diff
@@ -88,12 +78,11 @@ function StepOne({
                         className={st.input}
                         decimals
                     />
-
-                    <ErrorMessage
-                        className={st.error}
-                        name="amount"
-                        component="div"
-                    />
+                    {errors['amount'] && touched['amount'] ? (
+                        <div className="mt-3">
+                            <Alert>{errors['amount']}</Alert>
```

Comment:
It's probably worth (at some point in the future) making our `Input` component aware of the form and able to do self-registration + error display, but just something to remember for later not something to do now.